postgresql/manifests/backup-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+# PVC for backup volume
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: postgres-backup-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
Given the typical size of a database here, opting for a 10G default may lead to gobbling up more disk space than we need for the backup job.  Why not start with something tiny and then force people to specify a size suited for the particular job?